pom.xml

```diff
@@ -167,8 +166,35 @@ THE SOFTWARE.
       <version>1.0.2</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.openjdk.jmh</groupId>
+      <artifactId>jmh-core</artifactId>
+      <version>1.21</version>
+    </dependency>
+    <dependency>
+      <groupId>org.openjdk.jmh</groupId>
+      <artifactId>jmh-generator-annprocess</artifactId>
+      <version>1.21</version>
+    </dependency>
+    <dependency>
+      <groupId>org.reflections</groupId>
+      <artifactId>reflections</artifactId>
+      <version>0.9.11</version>
+    </dependency>
+    <dependency>
+      <groupId>io.jenkins</groupId>
+      <artifactId>configuration-as-code</artifactId>
+      <version>1.17</version>
+      <optional>true</optional>
+    </dependency>
+    <!-- Sample plugin for Configuration as Code support in benchmarks -->
```

Comment:
Would be better to use Matrix Auth which is already bundled.

Reply:
I will replace it. The jQuery dependency from Role strategy is causing the build to fail due to that 404 error.

Reply:
Done!